### PR TITLE
Add a docker-compose file for development and move over to postgres

### PR DESCRIPTION
**Pull request by fragosti**
The `docker-compose` file is so we don't have to manually run mesh, and now postgres in development. It does not use the `Dockerfile` use for deploying the API.

The reason for moving over to postgres is so that we can productionize this and point the API to an RDS instance in production.

### CODE REVIEW

File: .env_example [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Required
```
Comment (BMillman19) [inline .env_example]: do we want to provide sample values for these?

File: .env_example [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+NETWORK_ID=
+ETHEREUM_RPC_URL=
+
+# Optional
+MESH_ENDPOINT=
```
Comment (BMillman19) [inline .env_example]: Seems like we have a couple different naming conventions going on here, for example `ETHEREUM_RPC_URL` vs `MESH_ENDPOINT`

File: .env_example [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+FEE_RECIPIENT=
```
Comment (BMillman19) [inline .env_example]: `FEE_RECIPIENT_ADDRESS` ?

File: .env_example (hunk 1 above)
Comment (BMillman19) [reply .env_example]: Also might be good to provide some comments around what each defaults to w/o having to go look int `config.ts`

Comment (fragosti) [reply .env_example]: I would provide sample values if it weren't for the `ETHEREUM_RPC_URL` value. Are we ok with putting a valid value in there?

The defaults are mentioned in the README.

**Pull request merged**